include/PetriEngine/Colored/Expressions.h

```diff
@@ -370,6 +375,10 @@ namespace PetriEngine {
                 return res;
             }
 
+            bool is_single_color() const {
+                return false;
```

Comment:
Burde dette ikke være noget lignende:
```suggestion
                return _constituents.size() == 1 && _constituents[0].is_single_color();
```